api-reference/_hidden/dxChartCommonAnnotationConfig/x.md

```diff
@@ -5,8 +5,8 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Along with [y](/api-reference/_hidden/dxChartCommonAnnotationConfig/y.md '/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#y'), positions the annotation's center at a specific pixel coordinate. (0, 0) is the upper left corner of the chart.
+Along with [y](/api-reference/_hidden/dxChartCommonAnnotationConfig/y.md '{basewidgetpath}/Configuration/annotations/#y'), positions the annotation's center at a specific pixel coordinate. (0, 0) is the upper left corner of the chart.
```

Comment:
Why do you write "Along with"?

Reply:
An old fragment written before me. Should I apply the same corrections as in the comment above? Like that:

Used with [x](/api-reference/_hidden/dxChartCommonAnnotationConfig/x.md '{basewidgetpath}/Configuration/annotations/#x') to position the annotation's center at a specific pixel coordinate. (0, 0) is the upper left corner of the chart.

Reply:
Yes